src/views/LoginView.vue

```diff
@@ -73,7 +77,9 @@ img{
 .background-wrapper {
   background-color: #1F2833;
   border-radius: 5px;
-  
+  display: flex;
```

Comment:
the flexbox stuff is already defined in line 63

Reply:
should be good now